peps/pep-0747.rst

```diff
@@ -200,14 +200,20 @@ that represent a type assignable to ``str | None``::
   err1: TypeForm[str | None] = str | int  # Error
   err2: TypeForm[str | None] = list[str | None]  # Error
 
-By this same definition, ``TypeForm[Any]`` describes a type form object
-that represents the type ``Any`` or any type that is assignable to ``Any``.
-Since all types in the Python type system are assignable to ``Any``,
-``TypeForm[Any]`` describes the set of all type form objects
-evaluated from all valid type expressions.
+By this same definition, ``TypeForm[object]`` describes a type form object that
+represents the type ``object`` or any type that is assignable to ``object``.
+Since all types in the Python type system are assignable to ``object``,
+``TypeForm[object]`` describes the set of all type form objects evaluated from
+all valid type expressions.
```

Comment:
Everything said here applies to `TypeForm[object]`, so it seems simpler and clearer to simply describe `TypeForm[object]`, and avoid the tendency to wrongly conflate `Any` and `object`. To precisely describe `TypeForm[Any]` would require further discussion of gradual types, which is already present in the spec and doesn't seem necessary here.

Reply:
I did end up needing to describe `TypeForm[Any]`, in order to maintain that bare `TypeForm` has that meaning.

Reply:
I don't have a strong opinion about whether to use `TypeForm[object]` or `TypeForm[Any]` in this text. If you feel that `object` is clearer, I'm fine with the change.